8.gtex-interpret/README.md

```diff
@@ -28,7 +28,7 @@ Many cell-type signatures were implicated in both VAE models including skeletal
 However, a neutrophil signature was extracted from the VAE _k_ = 3 model and not the VAE _k_ = 2 model (**Panel A Below**).
```

Comment:
Not sure if you want to mention the caveat that the models aren't built upon each other so just because a signature was gained in feature 2 of model k=3 doesn't mean that it will be in the model with k>=3.  But maybe that's a safe assumption to make..?

Reply:
yes, exactly! This point is driven home by panel c. I will be sure to point this out explicitly, though. Great point